website/src/components/events/event-list-item.js

```diff
@@ -85,25 +85,37 @@ export default function EventListItem(props) {
     eventType = "Community Event"
   }
 
+  // Gatsby seems to hijack all <a> tags and, if they do not begin with http://
+  // or https://, appends the path to the domain instead of setting the URL as
+  // expected.  
+  let link = info.link
+  if (!link) link = info.where
+
+  // show link if there is a url available 
+  // TODO: is (.) dot sufficient to match any url? 
+  if (link.includes(".")) {
```

Comment:
I think checking for a period might be too permissive - an address might be spelled "King St. East", for instance, and in that case this would render that address as a URL.

For now can you just check for '.ca' and '.com' domains, and render those as `<a>` tags instead of `<Link>` components? I believe `<Link>` components are used for internal routing within the application - these links should always point to an external URL.

Reply:
As per your suggestion to match a link text:

```suggestion
  const domains = ["http", "twitch", "www", ".ca", ".com"]

  function checkLink(param) {
    if (!param.includes(" ") && domains.some(val => param.includes(val))) {
      if (!param.includes("http")) param = "http://" + param
      return param.includes("http") ? <Typography variant="a" component={Link} to={param}>{param}</Typography> : param;
    } else return param;
  }
```
And: 

` Where: {checkLink(info.where)}`